src/connector/SpectodaSimulatedConnector.ts

```
@@ -0,0 +1,654 @@
+
+  unselect(): Promise<null> {
+    logging.verbose(`unselect()`);
+
+    return new Promise(async (resolve, reject) => {
+      if (this.#connected) {
+        await this.disconnect();
+      }
+      await sleep(10); // unselect logic
+      this.#selected = false;
+      resolve(null);
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)